frontends/p4/p4-parse.ypp

```diff
@@ -813,6 +827,34 @@ actionRef
                                            $$ = new IR::ActionListElement(@2, $1, mce); }
     ;
 
+entry
+    : optAnnotations keysetExpression ':' actionInvocation
```

Comment:
Unfortunately, initializer is defined just an expression. I really think this should point to an invocation expression, i.e., we should at least have an action reference with (). As opposed to default_action which can also take actions without (). AFAIK, at least in P4_14, we distinguish between the two types (one is const and one is not). I think this is resolved in P4_16 by using the const keyword for default_action, but we still do not require the () for an action without parameters. I would prefer to have the () enforced in the grammar.